### ISSUE TITLE
Problem to generate the 3D visualization!

### ISSUE THREAD

**Issue by charlesneimog** [cac-t-u-s/om-sharp-users#44]
Hi Jean,

I am working in one path with the 3dc class. It works fine but, when I will visualize inside the 3dc, I am having this error.

![image](https://user-images.githubusercontent.com/31707161/128741815-ba12cb99-862b-4b88-b6d9-c48d0c53d5e3.png)

Is it some thing that I am doing wrong? 
[Patch.zip](https://github.com/cac-t-u-s/om-sharp-users/files/6956009/Patch.zip)

**Comment by j-bresson** [comment]
Hello Charles, I think this is a bug that I know, which happens when you try to move the 3D viewer (click+drag) when it was not yet initialized. For some reason, on Windows (and I think also on Linux) you need to initialize it manually by either double-clicking or pressing "Escape" with the 3D panel selected. Then the 3D curve should appear, and you should be able to visualize it.

**Comment by charlesneimog** [comment]
Thank you! It works now!

**Comment by j-bresson** [comment]
The crash will also be fixed in OM# 1.4 :)

**Comment by charlesneimog** [comment]
One request!!!

Maybe some thing that would be good to make is that the play mode to uses the GPU too. Because the visualization is very good and work very well, but the play doesn't work as it should (maybe my computer is not good enought).

**Comment by j-bresson** [comment]
Yes, I noticed the player is kind of laggy on Windows, but I'm affraid there's nothing I can reasonably do about it right now... :(

**Comment by j-bresson** [comment]
OM# 1.4: https://github.com/cac-t-u-s/om-sharp/releases/tag/v1.4
